src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.OSX.cs

```diff
@@ -160,7 +164,7 @@ private void CreateStreamAndStartWatcher()
                     _callback,
                     arrPaths,
                     Interop.EventStream.kFSEventStreamEventIdSinceNow,
-                    0.2f,
+                    0.0f,
```

Comment:
Can you help me understand what was going on here before?  It seems like now we just disable coalescing from the OS, was the coalescing causing problems?